### PR TITLE
Improve participants | map page.

### PR DESCRIPTION
**Pull request by hectoruch**
- Add function 3Dots to description on people involved modal.
- Link to Users on people modal.

### CODE REVIEW

**Comment by hectoruch** [comment]
- Create a event export view.
- Update carto dataset.
- Change texts.

**Comment by hectoruch** [comment]
- Update select 2 styles.
- Add hover effects.

select_green


select_blue

**Comment by hectoruch** [comment]
- Change hover effects on dropdowns menu.

image

- Delete "close layers" option on panel on map.

**Comment by hectoruch** [comment]
Ready to review @edbrett 😄

File: sites/all/themes/custom/ogp_theme/lib/layouts/countries/countriesPage.js [hunk 1]
```diff
@@ -95,7 +95,7 @@ function showCountriesPage() {
       countriesMap = initCountriesMap();
       $('.select-legend-dropdown').select2({
         minimumResultsForSearch: Infinity,
-        placeholder: 'All themes',
+        placeholder: 'All themesw',
```
Comment (edbrett) [inline sites/all/themes/custom/ogp_theme/lib/layouts/countries/countriesPage.js]: Is this correct?

**Review by edbrett — COMMENTED** [review]
Great. One comment

**Pull request merged**